generators/ariane

```diff
@@ -69,7 +69,12 @@ for components in ver_cmd_comp:
         top_module = args
         get_top_module = False
     elif args.endswith('.sv'):
-        sources += os.path.join(ariane_path, args) + ' '
+        # The Ariane file lists contains a reference to this file but the module itself
+        # isn't instantiated anywhere. Problematically, it contains references to a
+        # nonexistent interface, so it otherwise breaks elaboration. This hack can be
+        # removed if the problem gets fixed upstream.
+        if 'axi2apb_wrap.sv' not in args:
+            sources += os.path.join(ariane_path, args) + ' '
```

Comment:
I think we use the makefile provided by Ariane to generate this project here. Is this a problem that is maybe worth reporting in mainline Ariane? or is this specific to the runner wrapper here?

Reply:
Yeah, I suspect this is a problem upstream. I can file an issue with them to see what they think but in the meantime I think we should land this here.

Reply:
sure, thanks!

Reply:
https://github.com/openhwgroup/cva6/issues/672